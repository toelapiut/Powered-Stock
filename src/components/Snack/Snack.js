import React from 'react';
import PropTypes from 'prop-types';
import styles from './snack.module.css';


export const Snack = ({message}) => {
  return (
    <div className={styles.container}>
      <p className={styles.message}>{message}</p>
    </div>
  );
};

Snack.propTypes = {
  message: PropTypes.string.isRequired
};