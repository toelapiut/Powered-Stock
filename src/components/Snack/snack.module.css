.container{
    max-width:50em;
    width: auto;
    @apply bg-white py-3 px-8 rounded flex justify-center items-center mb-5;
}

.message {
    @apply font-bold text-sm text-koamaru
}