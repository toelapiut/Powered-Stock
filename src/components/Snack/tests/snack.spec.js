import  React from 'react';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Snack from '../';

test('should render correctly', () => {
  const wrapper = shallow(<Snack message={'You have a test message'}/>);
  expect(toJson(wrapper)).toMatchSnapshot();
});