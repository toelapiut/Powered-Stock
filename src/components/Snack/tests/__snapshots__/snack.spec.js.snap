// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="container"
>
  <p
    className="message"
  >
    You have a test message
  </p>
</div>
`;
